packages/actor-rdf-parse-shaclc/lib/ActorRdfParseShaclc.ts

```diff
@@ -32,10 +33,15 @@ export class ActorRdfParseShaclc extends ActorRdfParseFixedMediaTypes {
 
   public async runHandle(action: IActionRdfParse, mediaType: string, context: IActionContext):
   Promise<IActorRdfParseOutput> {
+    const prefixIterator = new PrefixWrappingIterator(
+      streamToString(action.data).then(str => parse(str, { extendedSyntax: mediaType === 'text/shaclc-ext' })),
```

Comment:
Not a requirement for this PR, but I just noticed that we're not `catch`-ing the rejection of this promise. So any error events in `action.data` will get lost here.
Could you look into adding a handler for this?

Reply:
@rubensworks - I think this is a non-issue since the [`PrefixWrappingIterator` extends the `WrappingIterator`](https://github.com/comunica/comunica/blob/2b6f38e3cf1446ea59a678ef327934cc74bfe310/packages/actor-rdf-parse-shaclc/lib/PrefixWrappingIterator.ts#L7) which in turn handles promises [by emitting an error event](https://github.com/RubenVerborgh/AsyncIterator/blob/3171521759194b3ce463f989d8253e35978ef690/asynciterator.ts#L2031).

Reply:
Ok, good, seems like a non-issue indeed then.